src/db/migrations/20230618104332_feature_flags.js

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ export function up (knex) {
+  return knex.schema
+    .createTable('feature_flags', table => {
+      table.string('name').primary().unique()
+      table.boolean('is_enabled').defaultTo(false)
+      table.text('description').default('')
+      table.specificType('dependencies', 'character varying(255)[]')
+      table.specificType('allow_list', 'character varying(255)[]')
+      table.specificType('wait_list', 'character varying(255)[]')
+      table.timestamp('created_at').defaultTo(knex.fn.now())
+      table.timestamp('modified_at').defaultTo(knex.fn.now())
```

Comment:
I was going to ask for these if you hadn't already included them, nice :) Please see my other comment re: soft-delete too.

Reply:
I haven't used it myself, but I saw that Knex also [has a `timestamps` function](https://knexjs.org/guide/schema-builder.html#timestamps) that supposedly automatically adds both of these?

Reply:
I think this way is more explicit (clearer)